src/middlewares/index.js

```diff
@@ -5,4 +5,5 @@ module.exports = {
   validate: require('./validate.middleware'),
   upload: require('./multer.middleware'),
   rateLimit: require('./rate-limit.middleware'),
+  loggingBot: require('./loggingBot.middleware'),
```

Comment:
theo fl các file đã khởi tạo trước đó
```suggestion
  loggingBot: require('./logging-bot.middleware'),
```